src/kontrol/__main__.py

```diff
@@ -856,9 +840,16 @@ def parse(s: str) -> list[T]:
         help='Create a new Forge project compatible with Kontrol',
         parents=[
             kontrol_cli_args.logging_args,
-            kontrol_cli_args.foundry_args,
         ],
     )
+    init.add_argument(
+        '--name',
+        dest='project_root',
+        default=Path.cwd(),
+        type=Path,
+        help='Name of the project to be initialized. If missing, the current directory is used.',
+    )
```

Comment:
Why `--name`? Does that match `forge init` behavior? I guess I would expect the same argument `--project-root` for this.

Also, I think defaults are supposed to be set in teh `*Options` classes, not here. @nwatson22 could say more.

Reply:
It looks ilke `forge init DIR` is more the way. So maybe we just make it an optional positional argument? https://stackoverflow.com/questions/4480075/argparse-optional-positional-arguments

```suggestion
    init.add_argument(
        dest='project_root',
        nargs='?',
        default=None,
        type=Path,
        help='Name of the project to be initialized. If missing, the current directory is used.',
    )
```

Reply:
Addressed in https://github.com/runtimeverification/kontrol/pull/507/commits/b466e8af92712414db09cf3952c9fb993df90d54.